### PR TITLE
fix(CORE/Commands): Renamed .character deleted old to .character deleted purge

### PR DESCRIPTION
**Pull request by peti446**
## CHANGES PROPOSED:
-  Renamed .character deleted old [keepDays] to .character deleted purge [keepDays]
-  Updated command description 

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2925

## TESTS PERFORMED:
1. Changed CharDelete.Method to 1 and CharDelete.KeepDays to > 0
2. Created and deleted character
3. Checked DB for the deleted character.
4. Executed .character deleted purge 0
5. Checked if deleted character is gone from the DB

## HOW TO TEST THE CHANGES:

1. Change CharDelete.Method to 1 and CharDelete.KeepDays to > 0
2. Create and delete a character
3. Check DB for the deleted character.
4. Execute .character deleted purge 0
5. Check if deleted character is gone from the DB

## KNOWN ISSUES AND TODO LIST:

- [ ] Update WIKI

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/scripts/Commands/cs_character.cpp [hunk 1]
```diff
@@ -38,7 +38,7 @@ class character_commandscript : public CommandScript
             { "delete",        SEC_CONSOLE,          true,  &HandleCharacterDeletedDeleteCommand,  "" },
             { "list",          SEC_ADMINISTRATOR,    true,  &HandleCharacterDeletedListCommand,    "" },
             { "restore",       SEC_ADMINISTRATOR,    true,  &HandleCharacterDeletedRestoreCommand, "" },
-            { "old",           SEC_CONSOLE,          true,  &HandleCharacterDeletedOldCommand,     "" },
+            { "purge",           SEC_CONSOLE,          true,  &HandleCharacterDeletedOldCommand,     "" },
```
Comment (Helias) [inline src/server/scripts/Commands/cs_character.cpp:41]: you could respect the indentation :P

Comment (peti446) [reply src/server/scripts/Commands/cs_character.cpp]: Yep sorry totaly missed that!

Comment (IntelligentQuantum) [reply src/server/scripts/Commands/cs_character.cpp]: HandleCharacterDeletedOldCommand -> HandleCharacterDeletedPurgeCommand

File: data/sql/updates/pending_db_world/rev_1597499028423474100.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1597499028423474100');
+
+UPDATE `command` SET `name` = 'character deleted purge', `help` = 'Syntax: .character deleted purge [#keepDays]\r \r Completely removes all characters from the database that where deleted more than #keepDays ago. If #keepDays not provided the used value from worldserver.conf option \'CharDelete.KeepDays\'. If \'CharDelete.KeepDays\' option is disabled (set to value 0) then this command can\'t be used without the specifying #keepDays parameter.' WHERE (`name` = 'character deleted old');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1597499028423474100.sql:3]:
```suggestion
UPDATE `command` SET `name` = "character deleted purge", `help` = "Syntax: .character deleted purge [#keepDays]\r\n Completely removes all characters from the database that where deleted more than #keepDays ago. If #keepDays not provided the used value from worldserver.conf option 'CharDelete.KeepDays'. If 'CharDelete.KeepDays' option is disabled (set to value 0) then this command can't be used without the specifying #keepDays parameter." WHERE (`name` = "character deleted old");
```

File: src/server/scripts/Commands/cs_character.cpp [hunk 3]
```diff
@@ -38,7 +38,7 @@ class character_commandscript : public CommandScript
             { "delete",        SEC_CONSOLE,          true,  &HandleCharacterDeletedDeleteCommand,  "" },
             { "list",          SEC_ADMINISTRATOR,    true,  &HandleCharacterDeletedListCommand,    "" },
             { "restore",       SEC_ADMINISTRATOR,    true,  &HandleCharacterDeletedRestoreCommand, "" },
-            { "old",           SEC_CONSOLE,          true,  &HandleCharacterDeletedOldCommand,     "" },
+            { "purge",         SEC_CONSOLE,          true,  &HandleCharacterDeletedPurgeCommand,     "" },
```
Comment (Kitzunu) [inline src/server/scripts/Commands/cs_character.cpp:41]: Please fix the spacing for the last "" },

**Comment by felwyn** [comment]
Fix is working! Command ".character deleted purge 0" is only availabe in worldserver.exe console NOT ingame!

**Comment by FrancescoBorzi** [comment]
@felwyn thanks for testing. About:

> Command `.character deleted purge 0` is only availabe in worldserver.exe console NOT ingame!

that's the expected default behaviour, because of: `SEC_CONSOLE`

Tip: this behaviour can be changed in the DB so you can use this command in game, if you'd like

**Pull request merged**